components/dnd_custom_time_picker_modal/index.ts

```diff
@@ -10,8 +11,9 @@ import {setStatus} from 'mattermost-redux/actions/users';
 import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
 
 import {GlobalState} from 'types/store';
+import {makeAsyncComponent} from 'components/async_load';
 
-import DndCustomTimePicker from './dnd_custom_time_picker_modal';
+const DndCustomTimePicker = makeAsyncComponent('DndCustomTimePicker', React.lazy(() => import('./dnd_custom_time_picker_modal')));
```

Comment:
Not something that needs to be done now, but I suppose we could eventually make this done automatically by the modal system somehow